### ISSUE TITLE
Customize icon, text and optional action button.

### ISSUE THREAD

**Issue by tomk9** [Almoullim/background_location#6]
The plugin works fine but there is no icon when I run example project.
![image](https://user-images.githubusercontent.com/15803067/64908856-50af8800-d705-11e9-8c27-36724e500616.png)
Could you add some customization options for the icon, icon color, text and make "Stop location service" action optional?

**Comment by 0xecute** [comment]
Any updates?

**Comment by 0xecute** [comment]
Hello.
If you are interested, I did the required modifications.
I had to fix the context because it was not working well with others plugins.
What I did is remove the notification completely.
I manage the notification directly in flutter with the plugin local_notifications
https://github.com/noinskit/flutter_local_notifications/ (Branch actions!)


Fork: https://github.com/0xecute/background_location

So what my pubspec.yaml looks like:
```
flutter_local_notifications:
    git:
      url: https://github.com/noinskit/flutter_local_notifications
      ref: actions
  background_location:
    git:
      url: https://github.com/0xecute/background_location
```

**Comment by Almoullim** [comment]
@0xecute Would be happy to accept a pull request if this change helps.

**Comment by 0xecute** [comment]
@Almoullim  Done.

**Comment by Francismb** [comment]
@Almoullim Can probably un-close this one.

**Comment by nicks258** [comment]
How to customized this?

**Comment by cormalenv** [comment]
@0xecute any chance of a pull request for this?

**Comment by Almoullim** [comment]
You can now use 
```dart
await BackgroundLocation.setNotificationTitle("Test Title");
await BackgroundLocation.startLocationService();
```

**Comment by Sly2024** [comment]
Hello,

I adapted the plugin on my computer to customise the notification on Android (but no action button)
I also managed to change the time interval
I'll be glad to help if someone tells me how to do it

**Comment by Almoullim** [comment]
@Sly2024 fork this repo, push your changes to the repo you now have under your name, then make a pull request from your repo to this one

**Comment by Almoullim** [comment]
Closed with #61 thanks to @Sly2024
